package.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
     "bower": "1.3.12",
+    "del": "1.1.1",
```

Comment:
It makes no sense to lock it at that version. What's wrong with 1.1.2 when it's released?